CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # 4.4.0 (2020-08-28)
 
 - Added a paging test fixture. [#209](https://github.com/blackbaud/skyux-lists/pull/209)
+- Fixed the repeater component to allow for turning reorderability on and off dynamically. [#208](https://github.com/blackbaud/skyux-lists/pull/208)
```

Comment:
What does `dynamically` mean here? Would `programmatically` be a better fit? Or just nothing at all?

Reply:
Looking at https://github.com/blackbaud/skyux-lists/issues/207, I wonder if we should just delete `dynamically` and end the sentence with `off`.

Reply:
Maybe? The bug was if you had reorderability on and then turned it off and back on. So it isn't really that it is done programmatically but more that it can switch back and forth.

Reply:
If it's just turning it off and back on, would it work to change this to:

```Fixed the repeater component to let users toggle reorderability on and off.```?

Reply:
That's fine. Updated.